charts/m4d-crd/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: m4d-crd
+description: M4D CustomResourceDefinitions
+version: 0.0.0
```

Comment:
Would you like to open an issue for a release process?  You might have an idea on how to set this version before a release without manual edits.

Reply:
no, automation, I want this to be a human decision as changing CRDs may not change each release (this is usually the case in other products)

Reply:
You mean that the version here is not the same as the M4D version?

Reply:
I see that fleet just fixes it as "0.0.0" so I guess no need to change ever?